### ISSUE TITLE
 Error icon does not work

### ISSUE THREAD

**Issue by dantehemerson** [bhilburn/powerlevel9k#903]
When I execute an incorrect order in the terminal, it does not show me the error icon, instead it shows me the correct icon.

Screenshot:
![Sreenshot error](https://image.ibb.co/dAXhXT/Screenshot_2018_07_11_23_58_53.png)
---
 I do not know how they are handling the errors, but I wanted it to work similar to this:

![Like this](https://image.ibb.co/czKUCT/Screenshot_2018_07_12_00_00_55.png)

This is in my `.zshrc` file:
```
##----------------- Powerline9k Customization
POWERLEVEL9K_MODE='awesome-fontconfig'
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(context status dir vcs)
POWERLEVEL9K_DISABLE_RPROMPT=true
POWERLEVEL9K_CONTEXT_TEMPLATE="\uF17C" # Linux icon  (%n@%m) default

POWERLEVEL9K_STATUS_CROSS=true
POWERLEVEL9K_STATUS_OK=true
##----------------

ZSH_THEME="powerlevel9k/powerlevel9k"
```

I'musing `Oh-my-zsh`
in = `Debian 9`
PL9K version = `v0.6.5`
font: `Roboto Medium for Powerline`, 
terminal: `Terminator`, 
zsh v. =   `v5.3.1`

**Comment by onaforeignshore** [comment]
On a side note - You can use `os_icon` instead of overriding the `context` segment. It will be faster as it won't need to determine the user.

**Comment by dritter** [comment]
Hi @dantehemerson ! 
Thanks for the report. I already build a PR to address this problem. @bhilburn I think we indeed need to build 0.6.6.. :/

@dantehemerson Could you try out #904 and see if that fixes the problem for you? That would be awesome.

**Comment by bhilburn** [comment]
@dantehemerson - Thanks for taking the time to report this issue, and then confirm that the fix in #904 worked for you! I have merged #904, and we will tag a new release, v0.6.6, per this discussion.
